examples/ScatterPlotWidget.py

```diff
@@ -33,19 +33,19 @@
 typeInds = np.random.randint(5, size=1000)
 data['type'] = np.array(strings)[typeInds]
 data['x_pos'] = np.random.normal(size=1000)
-data['x_pos'][data['type'] == 'Type-A'] -= 1
```

Comment:
What was the error that led to this change? I don't see any issues with numpy 1.11 or 1.13.